sale_promotion_rule/__manifest__.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com)
+# Benoît GUILLOT <benoit.guillot@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Promotion Rule",
+    "summary": "Module to manage promotion rule on sale order",
+    "version": "10.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Sale",
+    "website": "https://akretion.com",
+    'author': 'Akretion, ACSONE SA/NV, Odoo Community Association (OCA)',
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
```

Comment:
@lmignon if not any `external_dependencies` then remove it

Reply:
@nikul-serpentcs done